components/base/Table.tsx

```diff
@@ -40,21 +40,31 @@ const StyledTable = styled.div`
   tr:nth-of-type(even) {
     background-color: ${GREY_1};
   }
-  /* Top-left td cell */
-  tr:nth-of-type(2) td:nth-of-type(1) {
-    border-radius: 5px 0 0 0;
-  }
-  /* Top-right td cell */
-  tr:nth-of-type(2) td:nth-last-of-type(1) {
-    border-radius: 0 5px 0 0;
-  }
-  /* Bottom-right td cell */
-  tr:nth-last-of-type(1) td:nth-last-of-type(1) {
-    border-radius: 0 0 5px 0;
-  }
-  /* Bottom-left td cell */
-  tr:nth-last-of-type(1) td:nth-of-type(1) {
-    border-radius: 0 0 0 5px;
+
+  /* td border-radius */
```

Comment:
This is the only thing I really changed here. The border radius for the top row of the `<tbody>` wasn't working on some of the examples in the docs.